.github/workflows/pr-gate.yml

```diff
@@ -0,0 +1,70 @@
+name: pr-gate
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+
+  unit-test:
+    if: github.repository_owner == 'keikoproj'
+    runs-on: ubuntu-latest
```

Comment:
why only utuntu ?

Reply:
ubuntu is part of their hosted runners.
https://github.com/actions/virtual-environments#available-environments